n8n/helm/n8n/Chart.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v2
+name: n8n
+description: helm chart for n8n
+type: application
+version: 0.1.0
+appVersion: "v1.0.0"
+
+dependencies:
```

Comment:
this is fine just to get started with, but I'm guessing there's an existing n8n helm chart we should build on top of by including as a dep here?